File: lib/features/agenda/domain/entities/task.dart
class Task {
  final String id;
  final String userId;
  final String title;
  final String description;
  final String date;
  final String time;
  final String status;
  final String priority;

  const Task({
    required this.id,
    required this.userId,
    required this.title,
    required this.description,
    required this.date,
    required this.time,
    required this.status,
    required this.priority,
  });
}

File: backend/app/tasks/schemas/task.py
from pydantic import BaseModel


class TaskCreate(BaseModel):
    title: str
    description: str
    date: str
    time: str
    status: str = "Pendiente"
    priority: str = "Media"


class TaskUpdate(BaseModel):
    title: str
    description: str
    date: str
    time: str
    status: str
    priority: str


class TaskResponse(BaseModel):
    id: str
    user_id: str
    title: str
    description: str
    date: str
    time: str
    status: str
    priority: str